content/compose/bridge/usage.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Use the default Compose Bridge transformation
+description: Learn about and use the Compose Bridge default transformation
+keywords: compose, bridge, kubernetes
+---
+
+{{< include "compose-bridge-early-access.md" >}}
+
+Compose Bridge supplies an out-of-the box transformation for you Compose configuration file. Based on an arbitrary `compose.yaml` file, Compose Bridge produces:
+
+- A [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) so all your resources are isolated and don't conflict with resources from other deployments.
+- A [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) with an entry for each and every [config](../compose-file/08-configs.md) resource in your Compose application.
+- [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) for application services. This ensures that the specified number of instances of your application are maintained in the Kubernetes cluster.
+- [Services](https://kubernetes.io/docs/concepts/services-networking/service/) for ports exposed by your services, used for service-to-service communication.
+- [Services](https://kubernetes.io/docs/concepts/services-networking/service/) for ports published by your services, with type `LoadBalancer` so that Docker Desktop will also expose same port on the host.
```

Comment:
```suggestion
- [Services](https://kubernetes.io/docs/concepts/services-networking/service/) for ports published by your services, with type `LoadBalancer` so that Docker Desktop will also expose the same port on the host.
```